File: desktop/src/com/nopalsoft/dosmil/desktop/DesktopLauncher.java
package com.nopalsoft.dosmil.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.nopalsoft.dosmil.MainGame;
import com.nopalsoft.dosmil.handlers.GoogleGameServicesHandler;
import com.nopalsoft.dosmil.handlers.RequestHandler;

public class DesktopLauncher {

    static RequestHandler reqHandler = new RequestHandler() {

        @Override
        public void showRater() {

        }

        @Override
        public void loadInterstitial() {

        }

        @Override
        public void showInterstitial() {

        }

        @Override
        public void showFacebook() {

        }

        @Override
        public void showAdBanner() {

        }

        @Override
        public void shareOnTwitter(String message) {

        }

        @Override
        public void shareOnFacebook(String message) {

        }

        @Override
        public void removeAds() {

        }

        @Override
        public void hideAdBanner() {

        }

    };
    static GoogleGameServicesHandler gameHandler = new GoogleGameServicesHandler() {

        @Override
        public void signIn() {

        }

        @Override
        public boolean isSignedIn() {
            return false;
        }

        @Override
        public void getLeaderboard() {

        }

        @Override
        public void submitScore(long score) {

        }
    };

    public static void main(String[] args) {
        LwjglApplicationConfiguration cfg = new LwjglApplicationConfiguration();
        cfg.title = "2048 8Bit";
        cfg.width = 480;
        cfg.height = 800;

        new LwjglApplication(new MainGame(reqHandler, gameHandler), cfg);
    }
}

File: desktop/src/com/nopalsoft/dosmil/desktop/DesktopLauncher.kt
package com.nopalsoft.dosmil.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.nopalsoft.dosmil.MainGame
import com.nopalsoft.dosmil.handlers.GoogleGameServicesHandler
import com.nopalsoft.dosmil.handlers.RequestHandler

fun main(args: Array<String>) {

    val reqHandler = object : RequestHandler {
        override fun showRater() {}

        override fun loadInterstitial() {}

        override fun showInterstitial() {}

        override fun showFacebook() {}

        override fun showAdBanner() {}

        override fun shareOnTwitter(message: String) {}

        override fun shareOnFacebook(message: String) {}

        override fun removeAds() {}

        override fun hideAdBanner() {}
    }

    val gameHandler = object : GoogleGameServicesHandler {
        override fun signIn() {}

        override fun isSignedIn() = false

        override fun getLeaderboard() {}

        override fun submitScore(score: Long) {}
    }

    val cfg = LwjglApplicationConfiguration()
    cfg.title = "2048 8Bit"
    cfg.width = 480
    cfg.height = 800

    LwjglApplication(MainGame(reqHandler, gameHandler), cfg)

}